.github/ISSUE_TEMPLATE/6.docs-feedback.yml

```diff
@@ -9,13 +9,14 @@ body:
         Please provide a searchable summary of the issue in the title above ⬆️.
 
         Thanks for contributing by creating an issue! ❤️
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/material-ui/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Your issue may already have been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.
```

Comment:
```suggestion
      description: Your issue may have already been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.
```